disparate-flies/WorkforceManagement
# Ay - View Training Program

Please structure your pull requests this way:

- **Purpose of PR**
To fulfill ticket #21 - 
User should be able to view Training Programs
Given a user wants to view all training programs
When the user clicks the Training Programs item in the navigation bar
Then the user will see a list of all training programs that have not taken place yet

- **Issue number this request impacts** ------
#21 


- **Description of what was changed:**
I added a line to the SQL statement in the Create Method of the Training Controller so that when a user clicks the Training Programs item in the navigation bar, then the user will see a list of all training programs that have not takes place yet



- **Thorough list of steps to test**
1. git checkout into master
2. git fetch --all
3. git checkout ay-train
4. ls
5. start WorkforceManagement.sln
6. run project
7. in brower - click on Training Programs
8. make sure Training Programs are all in the future



if you don't do this you will be
**FIRED**
